src/js/controller/controller.js

```diff
@@ -428,9 +429,12 @@ Object.assign(Controller.prototype, {
                     startTime
                 });
                 _beforePlay = false;
-                if (_inInteraction(window.event)) {
+
+                if (_inInteraction(window.event) && !hasPrimedBeforePlay) {
                     _programController.primeMediaElements();
+                    hasPrimedBeforePlay = true;
```

Comment:
We should also update the value when playback starts outside of this check, because if play worked without an interaction, we don't want to prime again when toggles playback.